app/src/views/components/friends/request_list.tsx

```diff
@@ -34,7 +34,7 @@ export function RequestList({
         <FriendRequestItem
           name={item.name}
           key={item.uid}
-          testID={index.toString()}
+          testID={item.name}
```

Comment:
I think this is a better way of doing it as indexes can get messy in case of problems, so thank you !